== TEST (2019-04-16 14:59:33) 
-- CODE:
import math
x=pi
print(x)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    x=pi
NameError: name 'pi' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-04-16 14:59:48) 
-- CODE:
import math
x=(pi)
print(x)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    x=(pi)
NameError: name 'pi' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-04-16 15:00:07) 
-- CODE:
import math
x=(pi)
print(x)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    x=(pi)
NameError: name 'pi' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-04-16 15:00:21) 
-- CODE:
from math import *
x=(pi)
print(x)
-- OUTPUT:
3.141592653589793
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
